src/helpers/Functions.js

```diff
@@ -28,3 +28,13 @@ export const filterDateTime = (timestamp) => {
   }
   return dateTime;
 };
+
+export const capilizeFirstLetter = (name) =>
```

Comment:
rather than adding these irrelevant functions witht he date time funciton; please rename the file back to `DateTime` and create a new file called `StringParsers` or `NameParsers`